code/modules/mob/living/simple_animal/friendly/cat.dm

```diff
@@ -131,11 +131,11 @@ var/list/wizard_snakes
 		wizard_snakes = list()
 	wizard_snakes.Add(src)
 
-/mob/living/simple_animal/cat/snek/wizard/Die()
+/mob/living/simple_animal/cat/snek/wizard/death_untransmogrify()
```

Comment:
This will make it so that only wiz snakes with their `transmogged_from` set disappear on death.
*All* wiz snakes are supposed to disappear on death.